export-nightfall

```diff
@@ -0,0 +1,37 @@
+#! /bin/bash
+
+usage()
```

Comment:
Hey, can these scripts go into the test folder, or even maybe create a new folder just for scripts? They start piling up on the root folder

Reply:
Done